source/clear-linux/concepts/security.rst

```diff
@@ -0,0 +1,207 @@
+
+.. _security:
+
+|CL| Security 
+*********************
+
+|CL|, as it is distributed and maintained upstream, aims to make 
+systemic and layered security conscious decisions, which are both
+performant and practical. This security philosophy is rooted within
+the project's codebase and operating culture.
+
+Below are just some examples of how:
+
+
+.. contents:: :local:
+   :depth: 2
+
+
+
+Security in Updates
+===================
+
+The |CL| team believes in the potential benefits of 
+software security through open sourcing, incremental updates, and 
+rapidly resolving known security advisories.
+ 
+
+
+The latest Linux codebase
+--------------------------------
+
+The |CL| Project operates around the idea of open sourcing 
+ideas and innovations upstream through the Linux kernel and 
+using it to take advantage of the latest and greatest features. 
+This includes security-centered innovations adopted in the Linux upstream. 
+
+ 
+
+Automated Effective Updating
+----------------------------
+
+The |CL| content is updated at lightning pace, releasing full
+new versions of |CL| with incremental updates multiple times a day! 
+
+This `rolling release model`_ allows |CL| to consume the latest security
+fixes of software packages as soon as they become available. 
+There is no waiting for major or minor releases on |CL|. 
+
+But an update is not effective if it is just simply downloaded onto a system. 
+It needs to be obtained *AND* ensured that the new patched copy is being
```

Comment:
Line 50: The patched version must be loaded into memory rather than the older copy.